backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchBackendFactory.java

```diff
@@ -92,12 +94,13 @@
 
 			MultiTenancyStrategy multiTenancyStrategy = getMultiTenancyStrategy( name, propertySource );
 			ElasticsearchWorkFactory workFactory = new ElasticsearchStubWorkFactory( dialectSpecificGsonProvider, multiTenancyStrategy );
+			ElasticsearchWorkBuilderFactory workBuilderFactory = new ElasticsearchWorkBuilderFactoryImpl( dialectSpecificGsonProvider );
 
 			ElasticsearchAnalysisDefinitionRegistry analysisDefinitionRegistry =
 					getAnalysisDefinitionRegistry( backendContext, buildContext, propertySource );
 
 			return new ElasticsearchBackendImpl(
-					client, name, workFactory,
+					client, name, workFactory, workBuilderFactory,
```

Comment:
I suppose this is temporary, because you want to create one commit per work and you want everything to still compile in the meantime?

Ultimately (at the end of this PR) the work factory (the old one) will disappear, right?

Reply:
Exactly, eventually we will have only the latter. This is a transitory state that won't exist at the end of phase 1 of this issue.